test/spec/Feature/Query/RpcSpec.hs

```diff
@@ -1202,6 +1202,13 @@ spec actualPgVersion =
             `shouldRespondWith`
             [str|unnamed text arg|]
 
+        it "can insert xml directly" $
+          request methodPost "/rpc/unnamed_xml_param"
+            [("Content-Type", "text/xml"), ("Accept", "text/xml")]
+            [str|unnamed xml arg|]
+            `shouldRespondWith`
+            [str|unnamed xml arg|]
```

Comment:
Oh, didn't know this was valid XML. How about a more representative example? Like:

```suggestion
            [str|<note><from>John</from><to>Jane</to><message>Remember me</message></note>|]
            `shouldRespondWith`
            [str|<note><from>John</from><to>Jane</to><message>Remember me</message></note>|]
```

(Got it from https://www.w3schools.com/xml/xml_server.asp)

Reply:
Ok, done.